BedrockServer.cpp

```diff
@@ -711,7 +720,10 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
     // If a signal is caught on this thread, which should only happen for unrecoverable, yet synchronous
     // signals, like SIGSEGV, this function will be called.
     SSetSignalHandlerDieFunc([&](){
+        const string logMessage = addLogParams("CRASHING from BedrockServer::runCommand", command->getCommandLogParams());
```

Comment:
```suggestion
        SALERT("CRASHING from BedrockServer::runCommand", command->getCommandLogParams());
```

This does not work?

Reply:
It does, but I need that same message to be returned as a string in the `SSetSignalHandlerDieFunc` method. I'm setting it as the `logMessage` so we don't have to process the function addLogParams twice.

Reply:
Why don't we just do the `SALERT` in `_SSignal_StackTrace` and not need it here then?

Reply:
Because in 2 calls, we have SWARN, and in the other 2, we have SALERT. If they were all the same time, we could do that directly in `_SSignal_StackTrace`



https://github.com/Expensify/Bedrock/blob/eb0c80b31917c8d9d049c4ffd0502505e7b4db9f/BedrockServer.cpp#L487
https://github.com/Expensify/Bedrock/blob/eb0c80b31917c8d9d049c4ffd0502505e7b4db9f/BedrockServer.cpp#L724
https://github.com/Expensify/Bedrock/blob/eb0c80b31917c8d9d049c4ffd0502505e7b4db9f/BedrockServer.cpp#L587
https://github.com/Expensify/Bedrock/blob/eb0c80b31917c8d9d049c4ffd0502505e7b4db9f/BedrockServer.cpp#L678

Reply:
Since I do think we want them to be different, I left them to use the appropriate log level where they were instead of inside `_SSignal_StackTrace`

Reply:
I think SALERT everywhere is fine

Reply:
Doesn't SALERT create issues on GH?

Reply:
SALERT and SWARN create issues if they happened more than 7 times in 30 minutes, unless they have the string `ENSURE_BUGBOT`, which ensures they get created regardless of occurrence count.

Reply:
@flodnv @tylerkaraszewski what do you think?

Reply:
I'm fine either way, don't block on me.

Reply:
> We'll run the log line saying we're crashing after we've already sent the crash command. That doesn't make a lot of sense to me.

I don't think either order is obviously "better". I also don't think we need to long the same thing twice with ALERT and WARN levels. If we're crashing we can just ALERT once and that seems fine?

Reply:
OK, I'll change the PR to only use alert and we'll do the alert from `SSetSignalHandlerDieFunc`